tnoodlejs/readme.txt

```diff
@@ -0,0 +1,32 @@
+Note: This is the documentation for USING tnoodle in JavaScript. This is NOT on EDITING tnoodle
+in JavaScript.
+If you want a working example of tnoodle, then you can open demo.html (in the current directory) 
+in your browser, and you should see a couple of cube icons. When you click one of them, it 
+generates a scramble for that cube.
+
+The tnoodle.js file in the root directory (1 level above this) is what you need. So, the function
+for generating scrambles can be called like this:
+
+function puzzlesLoaded(puzzles) {
+  puzzles[333].generateScramble()
+          ^  This is where you should put the cube type.
+}
+
+Cube Types
```

Comment:
"Cube" isn't a very accurate word here, could you change it to "Puzzle"? Also, instead of listing all the puzzles here, could you just link to https://github.com/thewca/tnoodle/blob/master/scrambles/src/puzzle/puzzles?